docs/csharp/language-reference/keywords/snippets/InitExample1.cs

```diff
@@ -0,0 +1,11 @@
+﻿
```

Comment:
Consider adding all these new snippets under a new directory named `init`.

Reply:
Thanks for the suggestion. That would indeed be the prescribed pattern, but the _snippets_ folder at present consistently follows the pattern of locating all snippets for the _keywords_ folder directly under _keywords/snippets_, without separating snippets into subfolders named after articles. Rather than introduce an inconsistency here I think it better to follow the existing pattern.

Reply:
I agree with @tdykstra We agreed that both meet the standard. This is especially true in the language reference area for keywords and operators.